nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/TensorflowConversion.java

```diff
@@ -284,4 +289,36 @@ public TF_Graph loadGraph(byte[] content) {
 
         return graphC;
     }
+
+    public TF_Session loadSavedModel(String savedModelPath, TF_SessionOptions options, TF_Buffer runOptions,
+            String tag, String key, TF_Graph graph, Map<String, String> inputsMap, Map<String, String> outputsMap) {
+        TF_Buffer metaGraph = TF_Buffer.newBuffer();
+        TF_Status status = TF_Status.newStatus();
```

Comment:
Tr y to use the graph runner status that's already on the object.

Reply:
You want a new parameter for that?